packages/mock-block-dock/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@local/tsconfig/dom+jsx.json",
+  "compilerOptions": {
+    "skipLibCheck": true
+  },
```

Comment:
Keen to get thoughts on this, without this I was getting type errors when linting (in `node_modules/echarts/types/dist/shared.d.ts` )

cc @kachkaev

Reply:
What error is it? Can you fix it via `patch-package`? That’d be a preferred option IMO. When `skipLibCheck` is `true`, TypeScript replaces conflicting types with `any` and this increases the chances of runtime errors.

I’m not against `skipLibCheck`, but we should treat it as a last resort, if all other solutions are infeasible. ATM the only package without this flag is `site` – it was excluded from #585 to get the PR sooner than later.

Reply:
Have taken another look and think have fixed it with a patch in https://github.com/blockprotocol/blockprotocol/pull/605/commits/bd8d5a901ed5ac12faec925ae2fe41f4b6cfc82a. If this is preferable, I'll merge-in the patch instead of `skipLibCheck: true`?

Reply:
This is definitely preferable – this way we're not skipping the check of all libraries (safer), and we can easily tell where the issue is (because there's a patch) – vs `skipLibCheck: true` where we'll have no idea in 4 weeks why it's on without turning it off and seeing what breaks.